test/test_manifest_tsv.cpp

```diff
@@ -182,6 +182,9 @@ void test_multinode_manifest(bool shuffle)
     auto block       = manifest.next();
     auto block_node1 = manifest_node1.next();
     auto block_node2 = manifest_node2.next();
+    (void) block;
```

Comment:
This is to prevent warning that block variable is never used. I could add volalite to also prevent optimization of this just in case. I don't know this test well. Probably @AdrShv  knows more.